src/app/helptext/system/email.ts

```diff
@@ -26,6 +27,7 @@ export const helptext_system_email = {
 
   port: {
     placeholder: T("Mail Server Port"),
+    validation: [Validators.required, rangeValidator(0)],
```

Comment:
For validating ports you could use the max range of 65535 along with the min range of 0 (FTP service does this)